platform-includes/getting-started-config/unreal.mdx

```diff
@@ -21,3 +21,22 @@ The window can be accessed by going to editor's menu: **Project Settings > Plugi
 By default, the SDK is automatically initialized on application startup. Alternatively, the `Initialize SDK automatically` option can be disabled and in this case, explicit SDK initialization is required.
 
 To override SDK settings at runtime, use the `InitializeWithSettings` method of the `SentrySubsystem` class.
+
+```cpp
+FConfigureSettingsDelegate OnConfigureSettings;
+OnConfigureSettings.BindDynamic(this, &UMyGameInstance::ConfigureSentrySettings);
+
+void UMyGameInstance::ConfigureSentrySettings(USentrySettings* Settings)
+{
+	Settings->Dsn = TEXT("DSN");
+
+  // Add data like request headers, user ip address and device name,
+  // see https://docs.sentry.io/platforms/android/data-management/data-collected/ for more info
```

Comment:
Is it expected to link to the "Data Collected" page of Android here?
We don't have a "Data Collected" page for Unreal yet.
We could omit this line with the link for now, and add it when working on #13434.